src/i18n/translations.ts

```diff
@@ -5,6 +5,134 @@ export const languages = {
 
 export const defaultLang = 'de';
 
+export const serviceTranslations = {
+    'services.title': {
+        de: 'Unsere Dienstleistungen',
+        en: 'Our services',
+    },
+    'services.intro': {
+        de: 'Wir bieten Ihnen maßgeschneiderte Lösungen für Ihre Anforderungen. Profitieren Sie von der Multidisziplinarität unseres Teams. Wir bringen Kompetenzen aus Geographie und Informatik ein, um Probleme fachlich korrekt und effizient zu lösen. ',
+        en: 'We offer you customized solutions for your requirements. Benefit from the multidisciplinarity of our team. We bring together expertise in geography and computer science to solve problems correctly and efficiently.',
+    },
+    'services.advantages': {
+        de: 'Ihre Vorteile',
+        en: 'Your advantages',
+    },
+    'services.esg.heading': {
+        de: 'Bereitstellung von ESG-Indikatoren',
+        en: 'Provision of ESG indicators',
+    },
+    'services.esg.text': {
+        de: 'Wir bieten Ihnen die Möglichkeit, ESG-Indikatoren zu berechnen und bereitzustellen. Diese Indikatoren sind wichtig für die Bewertung von Unternehmen und deren Nachhaltigkeit.',
+        en: 'We offer you the possibility to calculate and provide ESG indicators. These indicators are important for the assessment of companies and their sustainability.',
+    },
+    'services.esg.textLong': {
+        de: 'Wir bieten Ihnen die Möglichkeit, ESG-Indikatoren zu berechnen und bereitzustellen. Diese Indikatoren sind wichtig für die Bewertung von Unternehmen und deren Nachhaltigkeit.  Hier fokussieren wir uns auf die Bereitstellung von Biodiversitätsindikatoren. Beispiele sind die Veränderung der Vegetation, die Veränderung der Landnutzung oder die Veränderung der Wasserverfügbarkeit. Zudem können wir für EUDR Entwaldungsindikatoren bereitstellen.',
```

Comment:
who do we address with this service? Can we say "YOUR company and YOUR sutainability"?

Reply:
0efd7c5a710776cef7d3aa562226df7741e70685